packages/backend-core/src/objectStore/utils.ts

```diff
@@ -71,7 +71,7 @@ async function processUrlAttachment(
   const fallbackFilename = path.basename(new URL(attachment.url).pathname)
   return {
     filename: attachment.filename || fallbackFilename,
-    content: response.body,
+    content: response.body!,
```

Comment:
How are we certain this `!` is correct? If we're not, it's best to fail early by adding a check above for `!response.body` and throwing an error.